### ISSUE TITLE
About the usage of validation data

### ISSUE THREAD

**Issue by fengh16** [zwt233/ALG#1]
Hi, I find that the labels of validation data are used in the code example. However, the labels of validation data are unknown unless they are labeled by the oracle in the active learning task. **Can the labels of validation data be used in this task?**

The details about the usages are as following:

1. In `train(epoch, model,record)`, `acc_val` is used to select the best model.
2. In `weight = 0.5*np.log(acc_val.cpu().numpy() /(1-acc_val.cpu().numpy() ))`, which is the weight of different models.

Thanks in advance!

**Comment by zwt233** [comment]
Thanks for your attention! Following the same setting as AGE (https://github.com/vwz/AGE/blob/master/algcn/train.py), the label of validation set can be used since it cannot directly provides supervision for model training.

**Comment by fengh16** [comment]
Thank you for your reply! Although it seems to be a common practice, **how should I determine the weights of models** (which is the $\alpha_i$ in Eq. (12) in your paper) when the validation data is not available?

**Comment by zwt233** [comment]
> Thank you for your reply! Although it seems to be a common practice, **how should I determine the weights of models** (which is the $\alpha_i$ in Eq. (12) in your paper) when the validation data is not available?

One possible method:
You can split part of the training set as validation set and get the corresponding $\alpha_i$. After that, you can retrain the GNN model on the whole training set and set its model weight as $\alpha_i$.

**Comment by fengh16** [comment]
Thanks!
